### PR TITLE
docs: CNCF onboarding

### PR DESCRIPTION
**Pull request by csatib02**
## Overview

- Added an adopters section: _Links to companies offering support are expected to go a page that at least mentions support of the project. This can either be the company homepage or a project-specific landing page._

- Updated the Slack links to point to: https://cloud-native.slack.com/archives/C078PHYK38W/

- Added statement to the landing page: "We are a Cloud Native Computing Foundation sandbox project."

- Added footer with trademark guidelines linking to the [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/) page.

### CODE REVIEW

**Review by akijakya — APPROVED** [review]
Nice job!

File: content/docs/community/_index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 8000
 
 If you have questions about Bank-Vaults or its components, get in touch with us on Slack!
 
-First, [register on the Outshift Slack](https://outshift.com/slack), then [visit our Slack channel](https://outshift.slack.com/messages/bank-vaults).
+First, [register on the CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf), then [visit the #bank-vaults Slack channel](https://cloud-native.slack.com/archives/C078PHYK38W/).
 
 You can also ask questions on [GitHub Discussions](https://github.com/orgs/bank-vaults/discussions). We share important updates also here.
```
Comment (keraron) [inline content/docs/community/_index.md:10]: We share important updates also here. > We also share important updates here.

File: content/docs/contributing.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Submitting pull requests and code changes is not the only way to contribute:
 
 - If you use Bank-Vaults in a production environment, add yourself to the [adopters list](https://github.com/bank-vaults/bank-vaults/blob/master/ADOPTERS.md).
 - Help new users with issues they may encounter on the GitHub Issues pages of our repositories
-- Participate in discussions in the [#bank-vaults community Slack channel](https://outshift.com/slack) or in [GitHub Discussions](https://github.com/orgs/bank-vaults/discussions)
+- Participate in discussions in the [#bank-vaults CNCF Slack channel](https://cloud-native.slack.com/archives/C078PHYK38W/) or in [GitHub Discussions](https://github.com/orgs/bank-vaults/discussions)
```
Comment (keraron) [inline content/docs/contributing.md:18]: I'd add periods at the end of every bullet.

File: content/docs/support.md [hunk 3]
```diff
@@ -5,6 +5,6 @@ weight: 1175
 
 ## Community support
 
-If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the Outshift Community Slack](https://outshift.slack.com/).
+If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```
Comment (keraron) [inline content/docs/support.md:8]: I'd insert "you can" in front of [open an issue].

The comma is not necessary before "or talk to us..."

**Review by keraron — COMMENTED** [review]
Just a few minor notes :)

File: content/docs/_index.md [hunk 4]
```diff
@@ -24,3 +24,7 @@ In addition, we also provide Helm charts for installing various components, as w
 
 - If you are new to **Bank-Vaults**, begin with the [getting started guide]({{<relref "installing">}}).
 - If you need help using Bank-Vaults, see the [Support page]({{< relref "/docs/support.md" >}}) for ways to contact us.
+
+## Support
+
+If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/bank-vaults/issues) or talk to us on the [CNCF Slack]({{< relref "/docs/support.md" >}}).
```
Comment (ramizpolic) [inline content/docs/_index.md:30]:
```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/bank-vaults/issues) or write us on [Slack]({{< relref "/docs/support.md" >}}).
```

File: content/docs/community/_index.md [hunk 5]
```diff
@@ -5,8 +5,8 @@ weight: 8000
 
 If you have questions about Bank-Vaults or its components, get in touch with us on Slack!
 
-First, [register on the Outshift Slack](https://outshift.com/slack), then [visit our Slack channel](https://outshift.slack.com/messages/bank-vaults).
+First, [register on the CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf), then [visit the #bank-vaults Slack channel](https://cloud-native.slack.com/archives/C078PHYK38W/).
```
Comment (ramizpolic) [inline content/docs/community/_index.md:8]:
```suggestion
First, [register on Slack](https://communityinviter.com/apps/cloud-native/cncf), then [visit the #bank-vaults Slack channel](https://cloud-native.slack.com/archives/C078PHYK38W/).
```

File: content/docs/support.md [hunk 6]
```diff
@@ -5,6 +5,6 @@ weight: 1175
 
 ## Community support
 
-If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the Outshift Community Slack](https://outshift.slack.com/).
+If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```
Comment (ramizpolic) [inline content/docs/support.md:8]:
```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or talk to us in the [#bank-vaults channel of the Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```

Comment (ramizpolic) [inline content/docs/support.md:8]:
```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue]([https://github.com/bank-vaults/bank-vaults/issues](https://github.com/bank-vaults/bank-vaults/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen)) or write us on [Slack]({{< relref "/docs/support.md" >}}).
```

Comment (ramizpolic) [inline content/docs/support.md:8]:
```suggestion
If you encounter problems while using Bank-Vaults that the documentation does not address, you can [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component or write us on [#bank-vaults channel on Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```

**Review by ramizpolic — APPROVED** [review]

**Pull request merged**